### PR TITLE
Test Button + Start/Finish testing reminder

### LINKED ISSUE #45: Start test automatically
We currently have a couple of users who have joined but have not completed the test. This mean that we need to make it easier to start the test. One solution would be to start the test once the user has joined automatically and not wait for the /test

### PR DESCRIPTION
**Pull request by TheLlamamarketer**
I have finished the test button, which works exactly like the /test command. The only issue is that the previous button has to manually get deleted every update of the bot.

The reminder will appear a week after the user joins the server and will explain how to start a test.

Closes #45

### CODE REVIEW

File: commands/test/test-command.ts [hunk 1]
```diff
@@ -93,6 +93,77 @@ const job = new cron.CronJob('0 0 * * * *', checkForFeedbackRequests); // checks
...
+    const actionRow = new ActionRowBuilder<ButtonBuilder>()
+        .addComponents(button);
+
+    const guildId = process.env.GUILD_ID;
+    if (!guildId) throw new Error('GUILD_ID is not defined in .env');
+
+    const guild: Guild | undefined = client.guilds.cache.get(guildId);
+    if (!guild) throw new Error('Guild not found');
+
+    (guild.channels.cache.get("1135557183845711983") as TextChannel).send({ components: [actionRow] });
```
Comment (johan-t) [inline commands/test/test-command.ts:111]: Is this the correct Id for the main server?

Comment (johan-t) [reply commands/test/test-command.ts:111]: Or what channnel Id is this?

**Review by johan-t — APPROVED** [review]
LGTM

File: commands/test/test-command.ts (hunk 1 above)
Comment (johan-t) [reply commands/test/test-command.ts:111]: Maybe add a comment for what channel it is to make it more readable

Comment (TheLlamamarketer) [reply commands/test/test-command.ts:111]: yes its the correct id, added comment

**Pull request merged**